crafty.py

```diff
@@ -324,6 +324,7 @@ def send_kill_command():
     console.info("Crafty Startup Procedure Complete")
     console.help("Type 'stop' or 'exit' to shutdown Crafty")
 
+
     if not daemon_mode:
```

Comment:
`git checkout origin/master crafty.py` will remove any changes to this file (or the GUI has an option to reset the file)